src/topology/continuous_function/algebra.lean

```diff
@@ -113,6 +113,27 @@ instance {α : Type*} {β : Type*} [topological_space α] [topological_space β]
   ..continuous_map.semigroup,
   ..continuous_map.has_one }
 
+@[to_additive]
+instance {α : Type*} {β : Type*} [topological_space α]
+  [locally_compact_space α] [topological_space β]
+  [monoid β]  [has_continuous_mul β] : has_continuous_mul C(α, β) :=
+⟨begin
+  refine continuous_of_continuous_uncurry _ _,
+  change continuous ((λ (x : β × β), x.1 * x.2) ∘
+    (λ (x : (C(α, β) × C(α, β)) × α), (x.1.1 x.2, x.1.2 x.2))),
```

Comment:
I think you can delete this `change` line.  Same for all the others.

Reply:
Great, thank you!